sample-apps/spark/build.gradle

```diff
@@ -0,0 +1,53 @@
+plugins {
+    id 'java'
+
+    // Apply the application plugin to add support for building a CLI application.
+    id 'application'
+}
+
+applicationName = "TraceEmitter"
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+repositories {
+    jcenter()
+    mavenCentral()
+    maven {
+        setUrl("https://oss.jfrog.org/libs-snapshot")
+    }
+    mavenLocal()
+}
+
+configurations {
+    extraLibs
+}
+
+jar {
```

Comment:
If you need a fatjar I recommend spring-boot plugin. Even when not using spring boot it works great for making a fatjar.

Though if we go with docker, we can just not generate fatjar, I don't think we need to support both patterns.

Reply:
Okay we will go with the docker approach you suggested.